static/styles/settings.css

```diff
@@ -471,14 +552,13 @@ input[type=checkbox].inline-block {
     width: auto !important;
 
     background: transparent;
-    border: 1px solid hsl(178, 100%, 40%);
-    color: hsl(178, 100%, 40%);
-
-    padding: 2px 5px;
     border-radius: 4px;
-
     margin-top: 14px;
     margin-left: 10px;
+    border: 1px solid #59a687;
```

Comment:
the reason we've been using e.g. `hsl(178, 100%, 40%);` to declare colors is so that they work sensibly in "night mode" without needing to add styles in `night_mode.css` for everything.  Try toggling that setting and you'll see what I mean.